foxy/distribution.yaml

```diff
@@ -3612,6 +3612,11 @@ repositories:
       url: https://github.com/ros2/rosidl_typesupport_gurumdds.git
       version: foxy
     status: developed
+  rosxbee:
+    doc:
```

Comment:
could you add a `source` entry?

Reply:
Thanks @ivanpauno I will make an entry and will update here.
Also will fill the license information in the package.

Reply:
@ivanpauno I have made both the changes, Let me know if I should anything else. I'm new to this so dont mind if I ask dumb questions